README.md

```diff
@@ -285,6 +285,18 @@ resp, err := rmqc.DeleteShovel("/", "a.shovel")
 
 ```
 
+### Operations on cluster name
+``` go
+// Get cluster name
+cn, err := rmqc.GetClusterName()
+// => ClusterName, err
+
+// Rename cluster
+resp, err := rmqc.SetClusterName(ClusterName{Name: "rabbitmq@rabbit-hole"})
```

Comment:
I wonder if this should take a string… or reusing the same struct is actually OK. In most cases I've seen a cluster name is not set to a previously fetched value.